lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -27,71 +27,47 @@ class WP_Font_Collection {
 		public $slug;
 
 		/**
-		 * The name of the font collection.
-		 *
-		 * @since 6.5.0
-		 *
-		 * @var string
-		 */
-		public $name;
-
-		/**
-		 * Description of the font collection.
-		 *
-		 * @since 6.5.0
-		 *
-		 * @var string
-		 */
-		public $description;
-
-		/**
-		 * Array of font families in the collection.
-		 *
-		 * @since 6.5.0
-		 *
-		 * @var array
-		 */
-		public $font_families;
-
-		/**
-		 * Categories associated with the font collection.
+		 * Font collection data.
 		 *
 		 * @since 6.5.0
 		 *
 		 * @var array
 		 */
-		public $categories;
+		private $data;
 
 		/**
-		 * Font collection json cache.
+		 * Font collection JSON file path or url.
 		 *
 		 * @since 6.5.0
 		 *
-		 * @var array
+		 * @var string
 		 */
-		private static $collection_json_cache = array();
```

Comment:
The static caching is removed in favor of storing the collection data in the class instance after it's been loaded and verified.